flow/project.py

```diff
@@ -1663,11 +2051,11 @@ def _fetch_scheduler_status(self, jobs=None, file=None, ignore_errors=False):
             scheduler_info = {sjob.name(): sjob.status() for sjob in self.scheduler_jobs(scheduler)}
             status = dict()
             print("Query scheduler...", file=file)
-            for job in tqdm(jobs,
-                            desc="Fetching operation status",
-                            total=len(jobs), file=file):
-                for group in self._groups.values():
-                    _id = group._generate_id(job)
+            for group in self._groups.values():
```

Comment:
Have we completely removed the progress bar?

Reply:
@b-butler We're now dealing with aggregates. We can introduce a new progress bar which it indicates the "fetching of group status"